src/native/corehost/test/nativehost/comhost_test.cpp

```diff
@@ -165,3 +196,24 @@ bool comhost_test::errorinfo(const pal::string_t &comhost_path, const pal::strin
 
     return true;
 }
+
+bool comhost_test::typelib(const pal::string_t &comhost_path, int count)
+{
+    HRESULT hr;
+
+    hr = load_typelib(comhost_path);
+    log_default_typelib_load(hr, std::cout);
+    if (FAILED(hr))
+        return false;
+
+    for (int i = 1; i < count + 1; i++)
+    {
+        pal::stringstream_t tlb_path;
+        tlb_path << comhost_path << '\\' << i;
```

Comment:
This is a place where having a link to that path syntax is helpful - the one about resource path with a trailing `\N`.